dockstore-webservice/src/main/resources/import.sql

```diff
@@ -43,3 +43,10 @@ ALTER TABLE tag ADD CONSTRAINT parentid_constraint FOREIGN KEY(parentid) REFEREN
 
 CREATE UNIQUE INDEX unique_collection_entry ON collection_entry_version USING btree (collection_id, entry_id) WHERE version_id IS NULL;
 CREATE UNIQUE INDEX unique_collection_entry_version ON collection_entry_version USING btree (collection_id, entry_id, version_id) WHERE version_id IS NOT NULL;
+
+ALTER TABLE user_profile DROP CONSTRAINT one_sign_in_method_by_profile;
+CREATE UNIQUE INDEX one_sign_in_method_by_profile ON user_profile USING btree (onlineprofileid, token_type) WHERE onlineprofileid IS NOT NULL;
```

Comment:
Doesn't this still require something like
`CREATE UNIQUE INDEX one_sign_in_method_by_profile_old ON user_profile USING btree (username, token_type) WHERE onlineprofileid IS NULL;` to enforce one profile per username before onlineprofileid is populated?